src/components/FormPlanning/index.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState } from 'react';
+
+import { Box, TextField, Select, InputLabel, MenuItem, FormControl, Button } from '@mui/material';

```

Comment:
This something we have to talk like grup. Import from `@mui/material` or `@mui/material/Box`

Reply:
Yeah, I think that we can talk this in the next meet